File: athrill-account-server/Libraries/Model/Exceptions/ATBusinessExceptionModel.cs
using AT.Common.Enum;
using System;
using System.Collections.Generic;
using System.Text;

namespace AT.Model.Exceptions
{
    public class ATBusinessExceptionModel
    {
        public List<ATBusinessExceptionMessageModel> Validations { get; set; }
        public string Message { get; set; }
    }
    public class ATBusinessExceptionMessageModel
    {
        public ATErrorLevel ErrorLevel { get; set; }
        public string Message { get; set; }
        public string TargetType { get; set; }
    }
}

File: athrill-account-client/projects/at-models/src/commons/exceptions/at-business-exception.model.ts
import { ATErrorLevel } from '../../constants/at-enums.enum';

export class ATBusinessExceptionModel
{
    public Validations : ATBusinessExceptionMessageModel[];
    public  Message : string;
}

export class ATBusinessExceptionMessageModel 
{
    public ErrorLevel : ATErrorLevel;
    public  Message : string;
    public  TargetType : string;
}